tests/karma/booknav-sidepanels/ia-bookmarks-list.test.js

```diff
@@ -0,0 +1,218 @@
+    ));
+    const response = await oneEvent(el, 'bookmarkColorChanged');
+
+    expect(response).to.exist;
+  });
+
+  // it('emits a custom event when the add bookmark button is clicked', async () => {
+  //   const el = await fixture(container(bookmarks));
+
+  //   setTimeout(() => (
+  //     el.shadowRoot.querySelector('.add-bookmark').click()
+  //   ));
+  //   const response = await oneEvent(el, 'addBookmark');
+
+  //   expect(response).to.exist;
+  // });
+
+  it('sets editedBookmark when an edit button is clicked', async () => {
+    const el = await fixture(container(bookmarks));
+    let prevState = el.editedBookmark;
+
+    el.shadowRoot.querySelector('li button').click();
+    await el.updateComplete;
+
+    expect(el.editedBookmark).not.to.equal(prevState);
+    expect(el.editedBookmark.page).to.equal(bookmarks[0].page);
+
+    // When clicking the same edit button while in edit mode, should toggle
+    // edit mode off and remove editedBookmark pointer
+    prevState = el.editedBookmark;
+    el.shadowRoot.querySelector('li button').click();
+    await el.updateComplete;
+
+    expect(el.editedBookmark).not.to.equal(prevState);
+    expect(el.editedBookmark.page).not.to.equal(bookmarks[0].page);
+  });
+
+  it('resets editedBookmark when saveBookmark callback called', async () => {
+    const el = await fixture(container(bookmarks));
+
+    [el.editedBookmark] = bookmarks;
+    await el.updateComplete;
+    el.saveBookmark({ detail: { bookmark: bookmarks[0] } });
+    await el.updateComplete;
+
+    expect(el.editedBookmark).not.to.equal(bookmarks[0]);
+    expect(el.editedBookmark).not.to.have.keys(['page', 'thumbnail', 'id']);
+  });
+
+  it('resets editedBookmark when deleteBookmark callback called', async () => {
+    const el = await fixture(container(bookmarks));
+
+    [el.editedBookmark] = bookmarks;
+    await el.updateComplete;
+    el.deleteBookmark({ detail: { id: bookmarks[0].id } });
+    await el.updateComplete;
+
+    expect(el.editedBookmark).not.to.equal(bookmarks[0]);
+    expect(el.editedBookmark).not.to.have.keys(['page', 'thumbnail', 'id']);
+  });
+
+  it('renders the bookmarks count', async () => {
+    const el = await fixture(container([bookmarks[0]]));
+    const bookmarksCount = await fixture(el.bookmarksCount);
+
+    expect(bookmarksCount.innerHTML).to.include('(1)');
+  });
+
+  it('does not render the bookmarks count when no bookmarks present', async () => {
+    const el = await fixture(container());
+    el.renderHeader = true;
+
+    await el.updateComplete;
+
+    expect(el.shadowRoot.querySelector('header small')).not.to.exist;
+  });
+
+  it('renders an optional header section', async () => {
+    const el = await fixture(container(bookmarks));
+    el.renderHeader = true;
+
+    await el.updateComplete;
+
+    expect(el.shadowRoot.querySelector('header')).to.exist;
+  });
+
+  // it('renders an optional add bookmark button', async () => {
```

Comment:
We should be able to skip these tests with `it.skip` or `xit` (not sure which one Karma uses). That's preferable to commenting out since it shows in the results as "X tests skipped".